testcases/android/master.jinja2

```diff
@@ -0,0 +1,107 @@
+{% extends "master.jinja2" %}
+
+{# timeouts #}
+{% set timeout_download = timeout_download|default(0) %}
+{% set timeout_deploy = timeout_deploy|default(25) %}
+{% set timeout_between_deploy_boot = timeout_between_deploy_boot|default(0) %}
+{% set timeout_boot = timeout_boot|default(10) %}
+{% set timeout_common_tests = timeout_common_tests|default(20) %}
+{% set timeout_tests = timeout_tests|default(10) %}
+{% set timeout_job_total = timeout_download + timeout_deploy + timeout_between_deploy_boot + timeout_boot + timeout_common_tests + timeout_tests %}
+
+{# urls #}
+{% set url_docker = url_docker|default("yongqinliu/linaro-android-docker:0.1") %}
+{% set url_test_defintions_repo = url_test_defintions_repo|default("https://github.com/Linaro/test-definitions.git") %}
+
+{% set job_name = job_name|default("base") %}
+
+{# variables needs to be converted to be used by master.jinja2 #}
+{% set lava_job_name = "lkft-android-" + KERNEL_BRANCH + '-' + BUILD_NUMBER + '-' + job_name %}
+{% set lava_device_type = lava_device_type %}
+{% set LAVA_JOB_PRIORITY = TEST_LAVA_JOB_PRIORITY %}
+{% set job_timeout = timeout_job_total %}
+
+{########## LAVA JOB DEFINITION ##############}
+{% block visibility %}
+  group:
+    - {{TEST_LAVA_JOB_GROUP}}
+{% endblock visibility %}
+
+{% block metadata %}
+  android.build: "{{BUILD_NUMBER}}"
+  android.name: "{{JOB_NAME}}"
+  android.url: "{{REFERENCE_BUILD_URL}}"
+  android.version: "{{ANDROID_VERSION}}"
+{% if ANDROID_VENDOR_FINGERPRINT is defined %}
+  android.build.vendor.fingerprint: "{{ANDROID_VENDOR_FINGERPRINT}}"
+{% endif %}
+{% if ANDROID_GSI_FINGERPRINT is defined %}
+  android.build.gsi.url: "{{ANDROID_GSI_URL}}"
+  android.build.gsi.fingerprint: "{{ANDROID_GSI_FINGERPRINT}}"
+{% endif %}
+{% if TUXSUITE_DOWNLOAD_URL is defined %}
+  tuxsuite.download.url: "{{TUXSUITE_DOWNLOAD_URL}}"
+{% endif %}
+  lkft.build.config: "{{LKFT_BUILD_CONFIG}}"
+  git branch: "{{KERNEL_BRANCH}}"
+  git repo: "{{KERNEL_REPO}}"
+  git commit: "{{KERNEL_COMMIT}}"
+  git describe: "{{KERNEL_DESCRIBE}}"
+  build-url: "{{BUILD_URL}}"
+  toolchain: "{{TEST_METADATA_TOOLCHAIN}}"
+  series: lkft
+{% block job_specific_metadata %}
+{% endblock job_specific_metadata %}
+{% endblock metadata %}
```

Comment:
a lot of the metadata are also project specific.

Reply:
will update, and move to the project lkft-android side